File: src/onreadyfyMe/Vehiculo.java
package onreadyfyMe;

import java.text.DecimalFormat;

public class Vehiculo implements Comparable<Vehiculo> {
	private String marca;
	private String modelo;
	private double precio;

	public String getMarca() {
		return marca;
	}

	public void setMarca(String marca) {
		this.marca = marca;
	}

	public String getModelo() {
		return modelo;
	}

	public void setModelo(String modelo) {
		this.modelo = modelo;
	}

	public double getPrecio() {
		return precio;
	}

	public void setPrecio(double precio) {
		this.precio = precio;
	}

	public Vehiculo() {
	}

	public Vehiculo(String marca, String modelo, double precio) {
		this.marca = marca;
		this.modelo = modelo;
		this.precio = precio;
	}

	public void mostrarTodosDatos() {
		DecimalFormat formatea = new DecimalFormat("###,###.00");
		System.out.println(
				"Marca: " + getMarca() + " // Modelo: " + getModelo() + " // Precio: $" + formatea.format(getPrecio()));
	}

	public String mostrarMarcaModelo() {
		String marcaModelo = getMarca() + " " + getModelo();
		return marcaModelo;
	}

	@Override
	public int compareTo(Vehiculo v) {
		if (this.precio > v.precio) {
			return -1;
		}
		if (this.precio < v.precio) {
			return 1;
		}
		return 0;
	}

}

File: Python/vehiculo.py
class Vehiculo():
    __marca = ''
    __modelo = ''
    __precio = 0

    def __init__(self, marca, modelo, precio):
        self.__marca = marca
        self.__modelo = modelo
        self.__precio = precio

    def get_marca(self):
        return self.__marca

    def set_marca(self, marca):
        self.__marca = marca

    def get_modelo(self):
        return self.__modelo

    def set_modelo(self, modelo):
        self.__modelo = modelo

    def get_precio(self):
        return self.__precio

    def set_precio(self, precio):
        self.__precio = precio

    def agregaVehiculo(self):
        vehiculos = []

    def __str__(self):
        return f'Marca: {self.get_marca()} // Modelo: {self.get_modelo()} // Precio: ${self.get_precio()}'

    def esLujo(self):
        pass
